Objects/rangeobject.c

```diff
@@ -171,6 +171,28 @@ range_dealloc(rangeobject *r)
     PyObject_Free(r);
 }
 
+static unsigned long
+get_len_of_range(long lo, long hi, long step);
+
+/// Return the length as a long, or -1 on error
+static inline long compute_range_length_long(PyObject *start, PyObject *stop, PyObject *step) {
```

Comment:
I'd recommend dropping the `inline` here; it likely has no significant effect. (Though if your benchmarks say otherwise, then I'll both (a) be very surprised, and (b) retract this comment.)

Reply:
Nit: please break the long line, for readability. (See PEP 7.)